mne/minimum_norm/tests/test_time_frequency.py

```diff
@@ -129,9 +130,152 @@ def test_tfr_with_inverse_operator(method):
         method=method,
         prepared=True,
     )
+    assert power.shape == phase_lock.shape
     assert np.all(phase_lock > 0)
     assert np.all(phase_lock <= 1)
     assert 5 < np.max(power) < 7
+    # fairly precise spot check that our values match what we had on 2023/09/28
+    if method != "eLORETA":
+        # check phase-lock using arbitrary index value since pl max is 1
+        assert_allclose(phase_lock[1, 0, 0], 0.576, rtol=1e-3)
+        # check power
+        max_inds = np.unravel_index(np.argmax(power), power.shape)
+        assert_allclose(max_inds, [0, 11, 135])
+        assert_allclose(power[max_inds], 6.05, rtol=1e-3)
+
+
+@testing.requires_testing_data
+def test_tfr_multi_label():
+    """Test multi-label functionality."""
+    tmin, tmax, event_id = -0.2, 0.5, 1
+
+    # Setup for reading the raw data
+    raw = read_raw_fif(fname_data)
+    events = find_events(raw, stim_channel="STI 014")
+    inv = read_inverse_operator(fname_inv)
+    inv = prepare_inverse_operator(inv, nave=1, lambda2=1.0 / 9.0, method="dSPM")
+
+    raw.info["bads"] += ["MEG 2443", "EEG 053"]  # bads + 2 more
+
+    # picks MEG gradiometers
+    picks = pick_types(
+        raw.info, meg=True, eeg=False, eog=True, stim=False, exclude="bads"
+    )
+
+    # Load condition 1
+    event_id = 1
+    events3 = events[:3]  # take 3 events to keep the computation time low
+    epochs = Epochs(
+        raw,
+        events3,
```

Comment:
nitpick: little gain in defining this variable as it's only used once. I realize this is copied from another test earlier in the file, but IMO it shouldn't have been done there either

```suggestion
    epochs = Epochs(
        raw,
        events[:3],  # take 3 events to keep the computation time low
```